docs/tasks/debug-application-cluster/audit.md

```diff
@@ -25,36 +27,111 @@ answer the following questions:
-    - URI as requested
-2. The response line containing:
-    - the unique id from 1
-    - response code
+If disabled, the legacy 1.6 behavior is preserved.
 
-Example output for user `admin` asking for a list of pods:
+## Audit Policy
+
+Audit policy is a document defining rules about what events should be recorded. It's passed
+to the [kube-apiserver][kube-apiserver] using the `--audit-policy-file` flag.
 
 ```
-2017-03-21T03:57:09.106841886-04:00 AUDIT: id="c939d2a7-1c37-4ef1-b2f7-4ba9b1e43b53" ip="127.0.0.1" method="GET" user="admin" groups="\"system:masters\",\"system:authenticated\"" as="<self>" asgroups="<lookup>" namespace="default" uri="/api/v1/namespaces/default/pods"
-2017-03-21T03:57:09.108403639-04:00 AUDIT: id="c939d2a7-1c37-4ef1-b2f7-4ba9b1e43b53" response="200"
+--audit-policy-file=/etc/kubernetes/audit-policy.yaml
+```
+
+If `AdvancedAuditing` is enabled and this flag is omitted, no events will be logged.
+
+The policy file holds rules that determine the level of an event. Known audit levels are:
+
+- `None` - don't log events that match this rule.
+- `Metadata` - log event metadata but not request or response body.
+- `Request` - log event metadata and request body but not response body.
+- `RequestResponse` - log event metadata, request and response bodies.
+
+When an event is processed it's compared against the list of rules in order.
+The first matching rule sets the audit level of the event. An example audit
+policy file:
+
+```yaml
+rules:
+  # Don't log watch requests by the "system:kube-proxy" on endpoints or services
+  - level: None
+    users: ["system:kube-proxy"]
+    verbs: ["watch"]
+    resources:
+    - group: "" # core API group
+      resources: ["endpoints", "services"]
+
+  # Don't log authenticated requests to certain non-resource URL paths.
+  - level: None
+    userGroups: ["system:authenticated"]
+    nonResourceURLs:
+    - "/api*" # Wildcard matching.
+    - "/version"
+
+  # Log the request body of configmap changes in kube-system.
+  - level: Request
+    resources:
+    - group: ""
+      resources: ["configmaps"]
+    # The empty string "" can be used to select non-namespaced resources.
+    namespaces: ["kube-system"]
+
+  # Log configmap and secret changes in all other namespaces at the Metadata level.
+  - level: Metadata
+    resources:
+    - group: "" # core API group
+      resources: ["secrets", "configmaps"]
+  - level: Request
+    resources:
+    - group: ""
+      resources: ["configmaps"]
+
+  # Log all other resources in core and extensions at the Request level.
+  - level: Request
+    resources:
+    - group: ""
+  - level: Request
+    resources:
+    - group: "extensions/v1beta1"
+
+  # A catch-all rule to log all other requests at the Metadata level.
+  - level: Metadata
 ```
 
-## Configuration
+A minimal audit policy file can be used to log all requests at the `Metadata` level:
+
+```yaml
+# Log all requests at the Metadata level.
+rules:
+- level: Metadata
+```
```

Comment:
The API references aren't up for 1.7 from what I can tell. Also I don't know if I'd go up there since internal APIs like the ImageReview webhook definition aren't published.

I can include the spec here.

https://kubernetes.io/docs/api-reference/v1.6/#pod-v1-core